src/main/kotlin/org/malachite/estella/services/UserService.kt

```diff
@@ -22,6 +22,12 @@ class UserService(
             throw UserNotFoundException()
         }
 
+    fun generatePassword(length:Int = 15):String {
+        val alphanumeric = ('A'..'Z') + ('a'..'z') + ('0'..'9')
```

Comment:
Jestem w 80 procentach pewny, że takie coś juz jest zaimplementowane - poszukaj, wydaje mi się, że w robocie to widziałem. Ale to też jest legitne, więc jak Ci się nie chce, to potraktuj ten komentarz edukacyjnie.

Reply:
Tu Baeldung który robił to dłużej w javie: https://www.baeldung.com/java-generate-secure-password.
Tu opisują że w Kotlinie nie ma generowania haseł: https://www.techiedelight.com/generate-pseudo-random-password-kotlin/
A tu generowanie randomowego string w kotlinie skąd wziąłem kod: https://stackoverflow.com/questions/46943860/idiomatic-way-to-generate-a-random-alphanumeric-string-in-kotlin

Reply:
OK, może mi się coś walnęlo. Możliwe, że chodziło, że same znaki już są, ale to możesz zostawić XDD trudno się kłócić ze źródłami XDDD